config.py

```diff
@@ -1,10 +1,56 @@
+import pygame
 import math as maths
+import time
 
 Width = 50  # C 50 How many squares wide the board is
 Height = 30  # C 30 Ditto but with height
 Size = 20  # C 20 The size of the sides of each square (in pixels)
```

Comment:
Rework this starting from the screen size in pixels. Set the game screen size -> divide by the number of squares to get the size per square. This will make your code look better by removing numerous `config.Size * config.Width + config.Edge` lines in the code. Also, nobody cares about the number of pixels per button, but some people might find the game screen too big or small.